builder/dockerfile/dispatchers_test.go

```diff
@@ -116,7 +114,7 @@ func TestFromScratch(t *testing.T) {
 	err := initializeStage(sb, cmd)
 
 	if runtime.GOOS == "windows" && !system.LCOWSupported() {
-		assert.Check(t, is.Error(err, "Windows does not support FROM scratch"))
+		assert.Check(t, is.Error(err, "Linux containers are not supported on this system"))
```

Comment:
As follow-up we should add a test that uses `platform=goos` and checks that it returns the original windows specific message.